packages/ui-tree-browser/src/TreeBrowser/TreeCollection/props.ts

```diff
@@ -38,12 +38,12 @@ import type { WithStyleProps, ComponentStyle } from '@instructure/emotion'
 import { CollectionProps, TreeBrowserBaseProps } from '../props'
 
 type TreeBrowserCollectionOwnProps = {
+  level: number
```

Comment:
on line 81 please make the PropType `.isRequired` too :)

Reply:
done